ios/Runner/Info.plist

```diff
@@ -41,5 +41,11 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSLocationWhenInUseUsageDescription</key>
+    <string>This app needs access to location when open.</string>
```

Comment:
I think we should make it more descriptive. e.g. _we use your location to find places near you. we never save your location history_

Reply:
Sure